source/config/__init__.py

```diff
@@ -315,6 +315,16 @@ def setStartOnLogonScreen(enable):
 		if execElevated(SLAVE_FILENAME, (u"config_setStartOnLogonScreen", u"%d" % enable), wait=True) != 0:
 			raise RuntimeError("Slave failed to set startOnLogonScreen")
 
+
+def openConfigurationDirectory():

```

Comment:
I don't think this method belongs in config, to me it seems more like a UI command.